scripts/load-breaches.js

```diff
@@ -0,0 +1,48 @@
+"use strict";
+
+const got = require("got");
+
+const AppConstants = require("../app-constants");
+const DBUtils = require("../db/utils");
+const pkg = require("../package.json");
+
+const HIBP_USER_AGENT = `${pkg.name}/${pkg.version}`;
+
+
+async function handleBreachesResponse(response) {
+  try {
+    const breachesJSON = JSON.parse(response.body);
+
+    for (const breach of breachesJSON) {
+      await DBUtils.createBreach(breach.Name, breach);
+    }
+  } catch (error) {
+    console.error(error);
```

Comment:
I don't think this will ever be reached, unless there is a JSON parsing error, since the `throw` was removed from `DBUtils.createBreach`.